examples/intersection_observer/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "intersection_observer"
+version = "0.1.0"
+authors = ["allan2"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dev-dependencies]
+wasm-bindgen-test = "0.3.18"
+
+[dependencies]
+seed = { path = "../../" }
+js-sys = { version = "0.3.51" }
+
+[dependencies.web-sys]
+version = "0.3.51"
+features = [
+    "IntersectionObserver",
+    "IntersectionObserverInit",
+    "IntersectionObserverEntry",
+]
```

Comment:
A missing new line, but it isn't a show-stopper for merging. Just FYI so you can configure your IDE.